lib/resty/etcd/serializers/json.lua

```diff
@@ -0,0 +1,6 @@
+local cjson = require("cjson.safe")
```

Comment:
add new file, you need update the file `lua-resty-etcd-master-0.1-0.rockspec`:

https://github.com/api7/lua-resty-etcd/blob/master/rockspec/lua-resty-etcd-master-0.1-0.rockspec